eservice/lib/libpdo_enclave/contract_request.cpp

```diff
@@ -227,9 +236,9 @@ ContractResponse ContractRequest::process_update_request(void)
     // the only reason for the try/catch here is to provide some logging for the error
     try
     {
-        GipsyInterpreter interpreter;
+        GipsyInterpreter *interpreter = worker_->interpreter_;
```

Comment:
This block needs the same changes as the block above (a getter for the interpreter which waits for ready, and a single call to interpreterDone() when finished